cmake/onnxruntime_framework.cmake

```diff
@@ -77,6 +77,7 @@ endif()
 
 if (onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS)
   target_compile_definitions(onnxruntime_framework PRIVATE DEBUG_NODE_INPUTS_OUTPUTS)
+  target_link_libraries(onnxruntime_framework sqlite3)
```

Comment:
>sqlite3 [](http://example.com/codeflow?start=46&length=7)

does SQLite need to be installed now? could it be made optional? do we need to add this to cgmanifests and the other third party library stuff?

maybe find_package() could be used too
[https://cmake.org/cmake/help/latest/module/FindSQLite3.html](https://cmake.org/cmake/help/latest/module/FindSQLite3.html)

Reply:
how about on WIndows?

Reply:
Yeah, let me add it to cgmanifests.

Reply:
I'm not sure about Windows. Yeah, it might be a good idea to use find_package.

Reply:
and possibly [https://github.com/microsoft/onnxruntime/blob/master/ThirdPartyNotices.txt](https://github.com/microsoft/onnxruntime/blob/master/ThirdPartyNotices.txt) too

Reply:
Can we avoid requiring the user to install sqlite3 even if he's not dumping the tensors in the DB?

Reply:
@pranavsharma Yes, I can make it an separate cmake option on top of the existing one.

Reply:
can we also add some instructions on how to get the required version of sqlite3? perhaps in a readme or a comment in the header file.